### DISCUSSION TITLE
CF firmware 2024.2 does not flash through cfclient

### DISCUSSION THREAD

**Discussion by NageshEranki** [bitcraze/discussions#1487] [Q&A]
Hi 

I am unable to flash the '2024.2' version of the crazyflie firmware through the cfclient(client is up-to-date) bootloader. Version '2023.11' however works just fine.

EDIT: I use the active marker deck. I first tried updating the firmware with a flow deck and the active marker attached -- through the cfclient bootloader. I'm not sure whether the active marker deck be attached during this process.

**Comment by knmcguire** [comment]
hi!

Not sure if that helps but you could try to remove the decks and try the update again? The new firmware contains a new bootloader for the nrf chip so it is a bit more complex.

To help us out could you provide information of the [cfclients](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/)'s debug tab (found under 'Help'>'about') while being connected to the crazyflie?
